editor/editor_node.cpp

```diff
@@ -2304,6 +2304,12 @@ void EditorNode::push_item(Object *p_object, const String &p_property, bool p_in
 	_edit_current();
 }
 
+void EditorNode::previous_item() {
```

Comment:
```suggestion
void EditorNode::edit_previous_item() {
```
Make more sense I think.

Reply:
Done!